training/general/src/environments/pytorch-1.12-py39-cuda11.6/spec.yaml

```
@@ -20,4 +20,4 @@ tags:
   Preview: ""
   Python: "3.9"
   DeepSpeed: "0.7.3"
-  Onnx Runtime: "1.12"
\ No newline at end of file
+  ONNXRuntime: "1.12"
```

Nit: Add trailing newline.
```suggestion
  ONNXRuntime: "1.12"

```